src/replica/replica_stub.h

```diff
@@ -435,6 +435,20 @@ class replica_stub : public serverlet<replica_stub>, public ref_counter
 #ifdef DSN_ENABLE_GPERF
     perf_counter_wrapper _counter_tcmalloc_release_memory_size;
 #endif
+
+    perf_counter_wrapper _counter_bulk_load_running_count;
+    perf_counter_wrapper _counter_bulk_load_downloading_count;
+    perf_counter_wrapper _counter_bulk_load_ingestion_count;
+    perf_counter_wrapper _counter_bulk_load_finish_count;
+    perf_counter_wrapper _counter_bulk_load_failed_count;
+    perf_counter_wrapper _counter_bulk_load_ingestion_reject_write_count;
+    perf_counter_wrapper _counter_bulk_load_recent_download_file_succ_count;
```

Comment:
What does 'recent' mean? I didn't found the differene to the one not named with 'recent'.

Reply:
As the comment below, the value of those perf_counters will be reset while getting the value, actually all of them it the `recent` value during this reporting round, not the total value.

Reply:
So how about to remove `recent` to reduce readers' confusion？

Reply:
done